contract/tests/test_contract.py

```diff
@@ -135,11 +135,11 @@ def _add_template_line(self, overrides=None):
         return self.env['contract.template.line'].create(vals)
 
     def test_add_modifications(self):
+        partner2 = self.partner.copy()
+        subtype = self.env.ref('contract.mail_message_subtype_contract_modification')
         self.contract.message_subscribe(
-            partner_ids=self.contract.partner_id.ids,
-            subtype_ids=self.env.ref(
-                'contract.mail_message_subtype_contract_modification'
-            ).ids
+            partner_ids=[partner2.id],
```

Comment:
Why are you preferring now this form instead of the more readable `partner_ids=partner2.ids`?